### PR TITLE
expands dv fuzzing example to work for any file

### PR DESCRIPTION
**Pull request by ablwr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1408,21 +1408,21 @@
...
+  <!-- Broken File -->
+  <span data-toggle="modal" data-target="#broken_file"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Make a broken file out of a perfectly good one">Broken file</button></span>
+  <div id="broken_file" class="modal fade" tabindex="-1" role="dialog">
     <div class="modal-dialog modal-lg">
       <div class="modal-content">
         <div class="well">
-          <h3>Makes a broken DV test file</h3>
-          <p>Modifies an existing, functioning DV file and intentionally breaks it for testing purposes.</p>
-          <p><code>ffmpeg -i <i>input_file</i>.dv -bsf noise -c copy <i>output_file</i>.dv</code></p>
+          <h3>Makes a broken video (or audio) test file</h3>
```
Comment (retokromer) [inline index.html]: Is `a broken audio and/or video test file` an idea? I mean: both on the same level.

File: index.html [hunk 2]
```diff
@@ -1408,21 +1408,21 @@
...
+          <p>Modifies an existing, functioning file and intentionally breaks it for testing purposes.</p>
+          <p><code>ffmpeg -i <i>input_file</i> -bsf noise=1 -c copy <i>output_file</i></code></p>
           <dl>
             <dt>ffmpeg</dt><dd>starts the command</dd>
-            <dt>-i <i>input_file</i>.dv</dt><dd>takes in a normal DV file</dd>
-            <dt>-bsf noise</dt><dd>sets bitstream filters for all to 'noise'. The <a href="https://www.ffmpeg.org/ffmpeg-bitstream-filters.html#noise">noise filter</a> intentionally damages the contents of packets without damaging the container</dd>
+            <dt>-i <i>input_file</i></dt><dd>takes in a normal file</dd>
+            <dt>-bsf noise=1</dt><dd>sets bitstream filters for all to 'noise'. Filters can be set on specific filters using syntax such as <code>-bsf:v</code> for video, <code>-bsf:a</code> for audio, etc. The <a href="https://www.ffmpeg.org/ffmpeg-bitstream-filters.html#noise">noise filter</a> intentionally damages the contents of packets without damaging the container. This sets the noise level to 1 but it could be left blank or any number above 0.</dd>
```
Comment (retokromer) [inline index.html]: please add `target="_blank"` in the link tag

**Review by retokromer — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -1408,21 +1408,21 @@
   </div>
   <!-- ends Play VGA SMPTE bars -->
 
-  <!-- Broken DV -->
-  <span data-toggle="modal" data-target="#broken_dv"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Make a broken DV file from a perfectly good one">Broken DV</button></span>
-  <div id="broken_dv" class="modal fade" tabindex="-1" role="dialog">
+  <!-- Broken File -->
```
Comment (retokromer) [inline index.html:1411]: please change `DV`> `File` also in the closing comment

**Review by retokromer — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ablwr) [reply index.html]: Better to just say test file with no clarifiers?

**Comment by ablwr** [comment]
Sorry about that, sleepy-lazy changes!

**Review by todrobbins — APPROVED** [review]
LGTM

**Review by retokromer — APPROVED** [review]

**Review by retokromer — APPROVED** [review]

**Pull request merged**